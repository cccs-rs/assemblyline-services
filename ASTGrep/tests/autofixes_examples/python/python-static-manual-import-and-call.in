__import__("builtins").exec("print('hel\"lo')")
__import__("builtins").exec(b"print('hel\"lo')")
__import__(b"builtins").exec("print('hel\"lo')")
__import__('base64').some_func("AAAA")
