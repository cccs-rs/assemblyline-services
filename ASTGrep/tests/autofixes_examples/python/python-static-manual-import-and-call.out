builtins.exec("print('hel\"lo')")
builtins.exec(b"print('hel\"lo')")
builtins.exec("print('hel\"lo')")
base64.some_func("AAAA")